diff --git a/node_modules/nomnoml/dist/nomnoml.js b/node_modules/nomnoml/dist/nomnoml.js
index 7b222b2..8717aff 100644
--- a/node_modules/nomnoml/dist/nomnoml.js
+++ b/node_modules/nomnoml/dist/nomnoml.js
@@ -2127,14 +2127,14 @@
         });
     }
     function compileFile(filepath, maxImportDepth) {
-        var fs = require('fs');
-        var path = require('path');
-        var directory = path.dirname(filepath);
-        var rootFileName = filepath.substr(directory.length);
-        function loadFile(filename) {
-            return fs.readFileSync(path.join(directory, filename), { encoding: 'utf8' });
-        }
-        return processImports(loadFile(rootFileName), loadFile, maxImportDepth);
+        // var fs = require('fs');
+        // var path = require('path');
+        // var directory = path.dirname(filepath);
+        // var rootFileName = filepath.substr(directory.length);
+        // function loadFile(filename) {
+        //     return fs.readFileSync(path.join(directory, filename), { encoding: 'utf8' });
+        // }
+        // return processImports(loadFile(rootFileName), loadFile, maxImportDepth);
     }
 
     var version = '1.4.0';
